magento/magento2
# Remove redundant model load

Removes unnecessary mode load which has a negative impact on performance


### Description (*)
The model load that has been removed is now redundant due to the `afterGetProductAttributes()` method in `sales-rule/Model/Plugin/QuoteConfigProductAttributes.php`. 

### Fixed Issues (if relevant)

### Manual testing scenarios (*)

1. ... Add a quote item associated with a sales rule to the cart
2. ...Ensure quote item is added with applicable rules applied

### Questions or comments

### Contribution checklist (*)
